### DISCUSSION TITLE
How to tie the data to secondary y-axis (line chart)?

### DISCUSSION THREAD

**Discussion by blazespirit** [nhn/tui.chart#702] [Q&A] [answered]
Hi, I want to have 2 series on line chart, 1 for the main y-axis on the left, and another 1 on the secondary y-axis on the right. I checked the documentation and know how to draw a secondary y-axis, but I cant find any explanation on how to "tie" the data to the secondary y-axis, both the data series will draw on the scale of main y-axis. Appreciate if anyone can shed some light on this

**Comment by jwlee1108** [comment, marked as answer]
@blazespirit 
Hello. I'm sorry for the late comment.
I'm sorry, but we don't have the feature you're talking about yet. 

You can only make a difference according to the chart type. Please refer to the following [link](https://github.com/nhn/tui.chart/blob/main/docs/en/common-axes.md#secondary-y-axis).

Actually, this function is one of the tasks we're going to do.

**Reply by blazespirit** [reply]
Thanks for the clarification 👍

**Comment by wisehackermonkey** [comment]
Thanks for the clarification, Id love to bump this feature! Ive got a line graph that uses it but im unable to take advantage of it! darn!
![image](https://user-images.githubusercontent.com/29699356/137037516-bc43c2b5-a287-422e-8524-921d3e0cb7b6.png)
